lib/mayaUsd/ufe/UsdAttribute.h

```diff
@@ -188,7 +188,11 @@ class UsdAttributeGeneric
 //! \brief Interface for USD token attributes.
 class UsdAttributeFilename
     : public Ufe::AttributeFilename
+#ifdef UFE_V4_FEATURES_AVAILABLE
```

Comment:
Copied verbatim from UsdAttributeGeneric at line 165.

Reply:
Right that was the only one not transitioned!!!